ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -553,6 +554,7 @@ const TypeConstraintMap& GetTypeConstraintMap() {
     type_constraint_map["Less"]["T"] = NGraphDTypes();
     type_constraint_map["LessEqual"]["T"] = NGraphDTypes();
     type_constraint_map["Log"]["T"] = NGraphNumericDTypes();
+    type_constraint_map["Log1p"]["T"] = NGraphNumericDTypes();
```

Comment:
the definition of "T" is real numbers here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/ops.pbtxt#L20768

We should use `NGraphRealDTypes` as `NGraphNumericDTypes` includes int types as well

Reply:
ok, will change this data type.